src/browser/HostInstaller.h

```diff
@@ -41,10 +41,10 @@ class HostInstaller : public QObject
     void updateBinaryPaths(const bool proxy, const QString location = QString());
 
 private:
-    QString     getTargetPath(const supportedBrowsers browser);
-    QString		getBrowserName(const supportedBrowsers browser);
-    QString     getPath(const supportedBrowsers browser);
-    QString     getInstallDir(const supportedBrowsers browser);
+    const QString     getTargetPath(const supportedBrowsers browser);
+    const QString		getBrowserName(const supportedBrowsers browser);
+    const QString     getPath(const supportedBrowsers browser);
+    const QString     getInstallDir(const supportedBrowsers browser);
```

Comment:
+1 for removing const from the return value, but please also make the member functions themselves const instead.

Reply:
Oh, and the enum type name should start with an uppercase letter and the parameters shouldn't be const, since they are passed by value anyway.